src/RNCamera.js

```diff
@@ -465,6 +467,12 @@ export default class Camera extends React.Component<PropsType, StateType> {
     }
   };
 
+  _onStatusChange = () => {
+    if (this.props.onStatusChange) {
+      this.props.onStatusChange({ cameraStatus: this.getStatus(), recordAudioPermissionStatus: this.getRecordAudioPermissionStatus() });
```

Comment:
```suggestion
      this.props.onStatusChange({ cameraStatus: this.getStatus(), recordAudioPermissionStatus: this.state.recordAudioPermissionStatus });
```

This should be fine!